xbmc/video/Teletext.cpp

```diff
@@ -32,11 +32,6 @@
 #include "filesystem/SpecialProtocol.h"
 #include "guilib/GraphicContext.h"
 
-#if HAVE_SDL_VERSION == 1
-#include <SDL/SDL_stdinc.h>
-#elif HAVE_SDL_VERSION == 2
-#include <SDL2/SDL_stdinc.h>
-#else
```

Comment:
btw, here https://www.libsdl.org/release/SDL-1.2.15/include/SDL_stdinc.h SDL_memcpy4 and SDL_memset4 are defined exacly the same. so I too dont see reason to keep using SDL_stdinc here. I've added a commit that does. can remove it later if you want me to